main.js

```diff
@@ -27,6 +37,13 @@ function createElementAttribute(name) {
   };
 }
 
+function createStringElementAttribute(name) {
```

Comment:
I imagine if the element were initialized in the html content (including the default attributes), this would work out. I was thinking about supporting creating the elements with their default attributes through javascript. For example, if you have a custom-red-box element, you could do something like:

```
var box = document.createElement('custom-red-box');
// This should create: <custom-red-box style="background:red"></custom-red-box>
box.getAttribute('style'); // contains default value "background: red"
```
So even without accessing through the easy accessors, it should exist. I'm not sure we can just get it for free.